recipes/doxygen/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "doxygen" %}
+{% set version = "1_8_12" %}
+{% set sha256 = "12142d0cb9dda839deb44a8aa16ff2f32fde23124a7c428c772150433c73f793" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-Release_{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/Release_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  features:
+      - vc9    # [win and py27]
+      - vc10   # [win and py34]
+      - vc14   # [win and py35]
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+
+test:
+  commands:
+    - test -f "${PREFIX}/bin/doxygen"   # [not win]
+
+about:
+  home: http://www.stack.nl/~dimitri/doxygen/index.html
```

Comment:
That's optional. Sadly it is also GPL2 so a bit redundant. Would skip it if I were you.